### DISCUSSION TITLE
Built with Vendure

### DISCUSSION THREAD

**Discussion by michaelbromley** [vendurehq/vendure#485] [General]
Here are some projects I know about that are already in production with Vendure:

* [CB Made In Italy](https://cbmadeinitaly.com/): Vendure + Vue Storefront
* [Racketworld.ch](https://racketworld.ch/shop/rackets): Vendure + Svelte + Sapper

Add your own below! 👇

**Comment by michaelbromley** [comment]
https://gomo.in.th/

> ReactJS on frontend, Storybook and Styled components for UI developments. Apollo GraphQL client to work with Shop API Client APIs. Created custom payment handlers using plugins for Backend, used custom fields on Product Entity. Thanks to Vendure support for active development :)

**Comment by rohanrajpal** [comment]
> Here are some projects I know about that are already in production with Vendure:
> 
> * [CB Made In Italy](https://cbmadeinitaly.com/): Vendure + Vue Storefront
> * [Racketworld.ch](https://racketworld.ch/shop/rackets): Vendure + Svelte + Sapper
> 
> Add your own below!

Hey, is Racketworld open source by any chance? We are also planning to use Svelte + Sapper and having an example would help a lot

**Reply by michaelbromley** [reply]
As far as I know no, but it was built by @oncode so he may be able to confirm.

**Reply by oncode** [reply]
@rohanrajpal Not open sourced yet, but if you need any help, you can ask me in the slack channel. 👍🏻

**Reply by rohanrajpal** [reply]
@oncode Awesome! Thanks :smile:

**Comment by michaelbromley** [comment]
### https://beerincans.com
Craft beer from Czechia, storefront built with Angular

From a developer who worked on the project:

> With the brand-new technology in cans filling and cans printing (no sleeves) we also decided for modern take on e-commerce.
As Angular developer I refused to take any ready-to-go PHP solution I knew from the past and decided for headless setup with Angular.
Only two solutions caught my interest - django-based "Saleor" and "Vendure". I've gone with Vendure and now I'm very happy I did.
Still in beta obviously it's not as polished and quite a few features I had to either implement myself or do without. But it's a small price for what you get back in terms of freedom of the headless, TS, dev-first experience.

> ...These advanced order flows and conditions is where Vendure shines. Also there is a bonus having TS on both sides - I was able to just take the condition as it is from BE and make UI component for it.

**Comment by martijnvdbrug** [comment]
I have posted mine here, for those interested: https://github.com/vendure-ecommerce/awesome-vendure
